hasura/metadata/databases/default/tables/public_app_metadata.yaml

```diff
@@ -0,0 +1,193 @@
+table:
+  name: app_metadata
+  schema: public
+object_relationships:
+  - name: app
+    using:
+      foreign_key_constraint_on: app_id
+insert_permissions:
+  - role: api_key
+    permission:
+      check: {}
```

Comment:
these permissions need constraints, both `user` and `api_key` can only be allowed to create records for apps they have permissions to